Step_Functions/Quantification_Alpha/Electrodes.m

```diff
@@ -36,6 +36,7 @@
 StepName = "Electrodes";
 Choices = ["F3,F4", "F3,F4,F5,F6,AF3,AF4", "F3,F4,P3,P4", "F3,F4,F5,F6,AF3,AF4,P3,P4,P5,P6,PO3,PO4"]; 
 Conditional = ["Cluster_Electrodes == ""no_cluster"" ", "NaN", "Cluster_Electrodes == ""no_cluster"" ", "NaN"]; 
+% Is it correct that "F3,F4,P3,P4" is "no cluster"? As it could be clustered as F3P3 and F4P4.
```

Comment:
yes because F is frontal and P is parietal.
These are included then as TWO localisations as they should include different processes and activity is not averaged across